src/universal/components/PrimaryButton/PrimaryButton.js

```diff
@@ -0,0 +1,77 @@
+/**
```

Comment:
I like what you did here, and am ok with this going in, when I take on #1766 I may combine what’s going on here with Button, or use this as a simple wrapper for Button. At any rate, enough talk about possible implementation, nice work on this!

Reply:
Thank you! That sounds great. Again did it this way to avoid disruption (and also because the `Button` implementation is pretty large...), but it'll be nice to do it The Right Way™️ when #1766 lands.